<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StopDlg</class>
 <widget class="QWidget" name="StopDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>182</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>182</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>182</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>30</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>130</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QPushButton" name="m_pBtnHome">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_pBtnResume">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="m_pBtnRepeat">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>42</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
